### ISSUE TITLE
[Screen Reader - Windows Powershell]: User is not able to navigate in edit area of powershell in scan mode of the narrator.

### ISSUE THREAD

**Issue by Chandresh1007** [microsoft/terminal#8477]
**User Experience:**
User who rely on screen reader will be affected if the the screen reader focus does not navigate to the edit area of windows powershell as the screen reader users will not able to navigate to the contents of the page .

**Test Environment:**
OS: Win 10 2004 Build (20262.1010)
App: Windows Terminal
App version: 1.4.3243.0

**Repro Steps:**
1. Open Windows terminal app.
2. Launch Narrator.
3. Turn on the scan mode of the narrator using "Caps+Space".
4. Now navigate using Up/Down arrow key.
5. Observe the issue.

**Actual Result:**
User is not able to navigate in edit area of powershell in scan mode of the narrator.

**Expected Result:**
In scan mode screen reader focus **should navigate to the edit area of powershell and read the text**. 

**Note:** NVDA browse mode is not working.

**MAS Reference:**
MAS4.2.10
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Bb24201d1-2276-4471-8183-e1e5353031ad%7D&cid=83352b15-6eaa-4fc1-bd5a-2d857a532986

[MAS4.2.10_Scan mode_not navigating to powershell window text.zip](https://github.com/microsoft/terminal/files/5635390/MAS4.2.10_Scan.mode_not.navigating.to.powershell.window.text.zip)

**Comment by carlos-zamora** [comment]
After watching the attached video, there's two issues here:
1. When you return focus back to the text area, you actually exited scan mode. That's why the arrow keys navigate through the history and the command prompt area. If you enter scan mode again, you should be able to navigate the buffer properly.
2. When you exited scan mode on accident, the screen reader wasn't reading the new commands injected in the command prompt. This is the same as https://github.com/microsoft/terminal/issues/8463. NVDA does not have this issue.

Can we close this issue as invalid? Again, if you enter scan mode when in the text area, scan mode should work just fine.

**Comment by Chandresh1007** [comment]
@carlos-zamora For 1st point, we are still not satisfied as user is still not able to access the text of the edit area in scan mode of the narrator even after turning it on again. 
Should we log this against narrator if the issue is not at the windows terminal end?

**Comment by carlos-zamora** [comment]
@Chandresh1007 I don't know how to record my screen with audio (sorry), but I know how to take a gif haha. I've attached it below. You can see the bounding rectangles for text that is being read. It seems to be working just fine with the repro steps provided above. Do you mind sending a video of this second attempt? 

![Narrator - Scan Mode Demo](https://user-images.githubusercontent.com/11050425/101382148-770e9580-386c-11eb-8be7-5e27f43b8376.gif)

**Comment by Chandresh1007** [comment]
@carlos-zamora Please find the attached video 
[Terminal_Scan mode issue.zip](https://github.com/microsoft/terminal/files/5659041/Terminal_Scan.mode.issue.zip)

**Comment by carlos-zamora** [comment]
> @carlos-zamora Please find the attached video
> [Terminal_Scan mode issue.zip](https://github.com/microsoft/terminal/files/5659041/Terminal_Scan.mode.issue.zip)

I see what's going on now. So, you entered Scan Mode, then used the up/down arrows to navigate to the next/previous item. At the 0:36 mark, Scan Mode is turned off for you. At 0:46, you enter Scan Mode again, but then you press up/down. Doing so does not navigate the text area, because, as mentioned before, this is used to navigate to the next/previous item. When you enter scan mode in the text area, you want to use these keybindings:

| Task | Command |
|--|--|
| Next/Previous Line | ctrl+up/down |
| Next/Previous Character | right/left |
| Next/Previous word | ctrl+right/left |

Source: [Narrator Keyboard Shortcuts](https://dequeuniversity.com/screenreaders/narrator-keyboard-shortcuts)

**Comment by Chandresh1007** [comment]
@carlos-zamora Yes, with the above mentioned keybindings, I am able to navigate text area in the scan mode. Thanks!

**Comment by carlos-zamora** [comment]
> @carlos-zamora Yes, with the above mentioned keybindings, I am able to navigate text area in the scan mode. Thanks!

Haven't been following what the correct procedure now is, but can we close this issue then?

**Comment by Chandresh1007** [comment]
Closing this issue as we are able to navigate the text area in scan mode using keybindings as mentioned above.
